tests/gradients/test_parameter_shift.py

```diff
@@ -311,6 +311,10 @@ def test_grad_recipe_parameter_dependent(self):
         shift rule"""
 
         class RX(qml.RX):
+            @property
+            def parameter_frequencies(self):
+                raise qml.operation.OperatorPropertyUndefined
+
             @property
             def grad_recipe(self):
                 # The gradient is given by [f(2x) - f(0)] / (2 sin(x)), by subsituting

```

Comment:
This is functionality we can't yet do in the gen-shift framework, but something that should be enabled once we have a UI to allow users to set shifts :)

Reply:
Yes exactly, definitely in-scope for this PR series :)